readme.txt

```diff
@@ -76,6 +76,10 @@ In your WordPress admin dashboard go to Appearance -> Widgets, drag the Astra :
 
 == Changelog ==
 
+= 1.2.10 =
+- Fix: Astra Social Profile: Repeater field not showing in widget block editor.
```

Comment:
@nileshc-bsf Need to add a doc link here.

Reply:
@premanshup - I will add it in the start infinity